package.json

```diff
@@ -10,7 +10,8 @@
     "asciidoctor-kroki": "0.15.4",
     "gulp": "~4.0",
     "gulp-connect": "~5.7",
-    "js-yaml": "~4.1"
+    "js-yaml": "~4.1",
+    "asciidoctor-external-callout": "~1.1.1"
```

Comment:
Shouldn't this be `1.0.0` ?

https://github.com/RayOffiah/asciidoctor-external-callout-js/tags